src/components/SegmentedControls/Slider.stories.tsx

```diff
@@ -32,8 +32,18 @@ Text.args = {
     ],
 };
 
-export const Number = SegmentedControlsTemplate.bind({});
-Number.args = {
+export const TextWithHugWidth = SegmentedControlsTemplate.bind({});
+TextWithHugWidth.args = {
+    items: [
+        { id: 'a', value: 'All' },
+        { id: 'b', value: 'Top 10' },
+        { id: 'c', value: 'Bottom 10' },
+    ],
+    hugWidth: true,
```

Comment:
![Screenshot 2023-05-15 at 16 55 48](https://github.com/Frontify/fondue/assets/35405438/7316fda8-22ce-478a-9e2e-38d3b14405f7)
Would be nice to add that property to argTypes so it doesn't appear as a textInput but a boolean toggle in Storybook controls.